Fix Highlighter, edits and build scrip

hardikj opened the pull request:
This pull requests

- fix the highlighter
- fix a small bugs to enable the edits again
- fix the build script
- fix column and row addition

hardikj commented on src/WickedGrid.js:
```diff
@@ -1938,7 +1938,7 @@ var WickedGrid = (function() {
 
       if (!doNotClearHighlighted) {
         this.highlighter
-            .cell(cell) //highlight the cell and bars
```
I can also do 

`.off()
.cell(cell)`

But set() has more control flows.
Let me know.

hardikj commented on build.js:
```diff
@@ -21,19 +21,19 @@ var fs              = require('fs')
 
 					'autoFiller',
-					'cellTypeHandlers',
+					'CellTypeHandlers',
```
do you want me to move all the files with Pascal Scale down below and all the files with camelCase up here?

robertleeplummerjr replied:
`LikeThis` for constructors `likeThis` for singletons.

hardikj commented on src/WickedGrid/Undo.js:
```diff
@@ -26,7 +26,7 @@ WickedGrid.Undo = (function() {
           before = (new WickedGrid.CellRange(cells)).clone().cells,
-          after = (typeof fn === 'undefined' ? (new WickedGrid.CellRange(fn(cells)).clone()).cells : before);
+          after = (typeof fn !== 'undefined' ? (new WickedGrid.CellRange(fn(cells)).clone()).cells : before);
 
```
The edits were failing because this guy was hiding in the code, took me some time to figure out 👍

robertleeplummerjr commented on src/WickedGrid/Undo.js:
```diff
@@ -26,7 +26,7 @@ WickedGrid.Undo = (function() {
       var self = this,
           before = (new WickedGrid.CellRange(cells)).clone().cells,
-          after = (typeof fn === 'undefined' ? (new WickedGrid.CellRange(fn(cells)).clone()).cells : before);
+          after = (typeof fn !== 'undefined' ? (new WickedGrid.CellRange(fn(cells)).clone()).cells : before);
```
Nice find.

robertleeplummerjr commented:
Can you write some tests that ensure this stays resolved in the future?  Currently tests are located: https://github.com/Spreadsheets/WickedGrid/tree/master/legacy-tests

But we do need some tlc there.

hardikj commented:
@robertleeplummerjr  Can you please have a fresh look?

I have fixed the tests and for the start enabled adding  of new rows and columns. I also added my commit which fixes adding new rows and columns.

Next, I will try to fix deleting columns and rows and will enable the tests for it as well.
